modules/exploits/multi/http/fortra_goanywhere_rce_cve_2023_0669.rb

```diff
@@ -21,6 +21,7 @@ def initialize(info = {})
         },
         'Author' => [
           'Ron Bowes', # Analysis and module
+          'Fryco'
```

Comment:
Should we add a comment here such as "Discoverer" or similar to explain this attribution?

Reply:
He technically discovered it, but never reported it, and didn't release his analysis until after mine.

We should link his blog, which I see Spencer already did based on our Slack convo: https://frycos.github.io/vulns4free/2023/02/06/goanywhere-forgotten.html

Reply:
Yeah, they're noted as the finder in the [CVE](https://www.cve.org/cverecord?id=CVE-2023-0669) details so that would make sense. Done.